packages/patternfly-4/react-core/src/helpers/util.test.js

```diff
@@ -93,13 +93,13 @@ test('sideElementIsOutOfView Returns NONE when in view', () => {
 describe('keyHandler works on ApplicationLauncher', () => {
   document.body.innerHTML = '<!doctype html><html><body></body></html>';
   const dropdownItems = [
-    <DropdownItem key=" 1" id="first" component="button">
+    <DropdownItem key=" 1" id="first" componentID="first-button" component="button">
       Action 1
     </DropdownItem>,
-    <DropdownItem key="2" id="second" component="button">
+    <DropdownItem key="2" id="second" componentID="second-button" component="button">
       Action 2
     </DropdownItem>,
-    <DropdownItem key="3" id="third" component="button" isDisabled>
+    <DropdownItem key="3" id="third" componentID="third-button" component="button" isDisabled>
```

Comment:
Can the dropdown provide a default ID here? 

It seems odd that users must provide an 'optional' property to have the dropdown function properly.

Reply:
This is just for the unit test.